website/source/guides/identity/authentication.html.md

```diff
@@ -210,9 +210,10 @@ path "auth/approle/login" {
 }
 
 # Read test data
-path "secret/mysql/*" {
+path "secret/data/mysql/*" {
```

Comment:
If the Vault server is running in **dev** mode, `kv-v2` is mounted at `secret`; otherwise, `kv-v1` by default.  So, I think the better solution is to add a comment instead:

```
# Read test data
# Set the path to "secret/data/mysql/*" if you are running kv-v2
path "secret/mysql/*" {
  capabilities = [ "read" ]
}
```

Or, 

```
# Read test data
# Set the path to "secret/mysql/*" if you are running kv-v1
path "secret/data/mysql/*" {
  capabilities = [ "read" ]
}
```

Reply:
Thanks, I think in most cases vault server isn't running in dev mode, updated as your first advice.